src/styles/modules/grid.scss

```diff
@@ -0,0 +1,10 @@
+.grid-one {
```

Comment:
bad module, why you are using it, grid-one has display flex, nothing related to grid, grid-two-item is the worth class I've ever seen, will chat about it later.